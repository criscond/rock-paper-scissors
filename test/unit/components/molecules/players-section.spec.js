import playersSection from '../../../../src/js/components/molecules/players-section';
describe('Players Section', () => {

	it('should create a section', () => {
		expect(playersSection()).toMatchSnapshot();
	});
});
