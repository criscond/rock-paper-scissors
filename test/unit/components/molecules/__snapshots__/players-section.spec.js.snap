// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Players Section should create a section 1`] = `
<section>
  <span
    class=""
  >
    Select the opponents: 
  </span>
  <select
    id="gameType"
  >
    <option
      value="1"
    >
      Human vs Computer
    </option>
    <option
      value="2"
    >
      Computer vs Computer 
    </option>
  </select>
</section>
`;
